### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by guillermooo**

### CODE REVIEW

File: lib/src/common.dart [hunk 1]
```diff
@@ -15,7 +15,7 @@ import '../stagehand.dart';
 
 const int _RUNE_SPACE = 32;
 
-List<TemplateFile> decodeConcanenatedData(List<String> data) {
+List<TemplateFile> decodeConcatenatedData(List<String> data) {
```
Comment (devoncarew) [inline lib/src/common.dart]: I know this is in a private file, but I'd feel more comfortable if we kept a deprecated version of the method around, that just forwarded to the correct name:
```
@Deprecated('see decodeConcanenatedData')
List<TemplateFile> decodeConcanenatedData(List<String> data) =>
    decodeConcatenatedData(data);
```

**Comment by devoncarew** [comment]
Thanks! One comment. Can you also add yourself to the AUTHORS file in the PR?

**Comment by guillermooo** [comment]
Done!

**Comment by devoncarew** [comment]
Thanks!

**Pull request merged**